js/pages/pathways/components/browser.js

```diff
@@ -0,0 +1,109 @@
+define([
+	'knockout',
+	'text!./browser.html',
+	'../const',
+	'appConfig',
+	'../PathwayService',
+	'webapi/AuthAPI',
+	'providers/Component',
+	'utils/CommonUtils',
+	'utils/DatatableUtils',
+	'less!./browser.less'
+], function (
+	ko,
+	view,
+	constants,
+	config,
+	PathwayService,
+	authApi,
+	Component,
+	commonUtils,
+	datatableUtils
+) {
+	class PathwaysBrowser extends Component {
+    constructor(params) {
+      super(params);      
+      this.loading = ko.observable(false);
+      this.config = config;
+      this.analysisList = ko.observableArray();
+      this.isAuthenticated = authApi.isAuthenticated;
+      this.canRead = ko.pureComputed(() => {
+        return (config.userAuthenticationEnabled && this.isAuthenticated() && authApi.isPermittedReadIRs()) || !config.userAuthenticationEnabled;
```

Comment:
`this.isAuthenticated()` is redundant - this check is encapsulated in `AuthAPI.isPermitted`